test/1.4/errors/dml-1.2.dml

```diff
@@ -2,5 +2,5 @@
   © 2021-2023 Intel Corporation
   SPDX-License-Identifier: MPL-2.0
 */
-/// WEXPERIMENTAL
+/// WARNING WEXPERIMENTAL
```

Comment:
oh, this is apparently not used by the test system, so better to remove the comment. (can be pulled in by `/// SCAN-FOR-TAGS` in the main file, but I suppose there's no warning there since the test does succeed today)